_posts/2024-06-05-trino-fest-last-call.md

```diff
@@ -0,0 +1,78 @@
+---
+layout: post
+title: "One busy week to go to Trino Fest 2024"
+author: "Manfred Moser"
+excerpt_separator: <!--more-->
+image: /assets/blog/trino-fest-2024/announcement-banner.png
+show_pagenav: false
+---
+
+This week has surely started off with [a big
+bang](https://www.snowflake.com/blog/introducing-polaris-catalog/) and [another
+boom](https://www.databricks.com/blog/databricks-tabular) in the data platform
```

Comment:
I think you should try to be a little more descriptive with this: maybe "Started off with a big band and another boom in the data platform world. Snowflake introduced the Polaris catalog for Iceberg, and Databricks purchased Tabular." then use the more descriptive things for links. That way people don't necessarily _have_ to click to understand what you're talking about.

Reply:
Done